packages/NodeNameResolver/NodeNameResolver.php

```diff
@@ -52,7 +52,7 @@ public function isNames(Node $node, array $names): bool
     /**
      * @param Node|Node[] $node
      */
-    public function isName(Node | array $node, string $name): bool
+    public function isName(Node | array $node, string $name, bool $strict = false): bool
```

Comment:
The param name "strict" can mean anything.
Better would be to have 2 methods, one with "insensitive" in the name.